nextcord/client.py

```diff
@@ -2023,33 +2039,171 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         """
         return self._connection.get_application_command(command_id)
 
-    def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+    def __actual_get_application_command_from_signature(
+        self,
+        *,
+        type: Union[int, ApplicationCommandType],
+        qualified_name: str,
+        guild_id: Optional[int],
+        search_locales: bool = False,
+    ) -> Optional[Union[BaseApplicationCommand, SlashApplicationSubcommand]]:
         """Gets a locally stored application command object that matches the given signature.
 
+        .. versionadded:: 2.0
+
+        .. versionchanged:: 2.3
+            it's no longer possible to pass ``None`` to the ``name`` parameter.
+
+        .. versionchanged:: 2.3
+            Renamed the ``name`` parameter to ``qualified_name``.
+
+        .. versionchanged:: 2.3
+            Renamed the ``cmd_type`` parameter to ``type``.
+
+        .. versionchanged:: 2.3
+            Added the ``search_locales`` keyword argument.
+
+        .. versionchanged:: 2.3
+            Group/subcommand commands can now be retrieved with this method.
+
+        .. versionchanged:: 2.3
+            Changed the signature from ``(qualified_name, type, guild_id, *, search_locales)`` to ``(*, type, qualified_name, guild_id, search_locales)``.
+            All parameters are now keyword-only.
+
         Parameters
         ----------
-        name: :class:`str`
-            Name of the application command. Capital sensitive.
-        cmd_type: Union[:class:`int`, :class:`ApplicationCommandType`]
+        type: Union[:class:`int`, :class:`ApplicationCommandType`]
             Type of application command.
+        qualified_name: :class:`str`
+            Full name of the application command. Capital sensitive.
+            Subcommand commands must be separated by a space, E.g, ``parent group subcommand``.
         guild_id: Optional[:class:`int`]
             Guild ID of the signature. If set to ``None``, it will attempt to get the global signature.
+        search_locales: :class:`bool`
+            Whether to also search through the command's name locals. Defaults to ``False``.
 
         Returns
         -------
-        command: Optional[:class:`BaseApplicationCommand`]
-            Application Command with the given signature. If no command with that signature is
-            found, returns ``None`` instead.
+        command: Optional[:class:`BaseApplicationCommand`, :class:`SlashApplicationSubcommand`]
+            Application Command with the given signature. If found else ``None`` is returned instead.
```

Comment:
```suggestion
            Application Command with the given signature. If no command with that signature is found, ``None`` is returned instead.
```
The phrasing here didn't make sense and the older version didn't need changing anyways

Reply:
Oh, i was going with a dpy like style 🤷🏼